src/renderer/store/index.js

```diff
@@ -0,0 +1,53 @@
+import { create } from "zustand";
+
+const createClientStatusSlice = (set, get) => ({
+  clientStatus: {
+    isSubmitted: false,
+  },
+  setClientStatus: (setValue) =>
+    set((state) => ({
+      clientStatus: { ...state.clientStatus, ...setValue },
+    })),
+  getClientStatus: () => get().clientStatus,
+});
+
+const initialOrder = {
+  action: "",
+  attachmentName: "",
+  attachmentUrl: "",
+  sourcePath: "",
+  executionPath: "",
+  editingName: "",
+};
+
+const createOrderSlice = (set, get) => ({
+  order: initialOrder,
+  updateOrder: (setValue) =>
+    set((state) => ({
+      order: { ...state.order, ...setValue },
+    })),
+  getOrder: () => get().order,
+  clearOrder: () =>
+    set(() => ({
+      order: { ...initialOrder },
+    })),
+});
+
+const createOrdersSlice = (set, get) => ({
+  orders: [],
+  addOrder: (newOrder) =>
+    set((state) => ({ orders: [...state.orders, newOrder] })),
+  deleteOrder: (targetIndex) =>
+    set((state) => ({
+      orders: state.orders.filter((value, index) => index !== targetIndex),
+    })),
+  getOrders: () => get().orders,
+});
+
+const usePackageStore = create((set, get) => ({
+  ...createClientStatusSlice(set, get),
+  ...createOrderSlice(set, get),
+  ...createOrdersSlice(set, get),
+}));
```

Comment:
여기에서만 화살표 함수 표현을 () => () 형태를 해주셨는데 특별한 이유라도 있으신가요??? 🤔

Reply:
네, 수보님! 화살표 함수에서 객체를 반환할 시에는 소괄호로 감싸서 내보내야 하기 때문입니다!
[MDN 화살표 함수](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions#function_body) 관련 설명 중 하단으로 내려가시면 설명이 있습니다. 

> Returning object literals using the expression body syntax (params) => { object: literal } does not work as expected.
body syntax (params) => { object: literal }를 사용하여 개체 리터럴을 반환하는 것이 예상대로 작동하지 않습니다.

> This is because JavaScript only sees the arrow function as having an expression body if the token following the arrow is not a left brace, so the code inside braces ({}) is parsed as a sequence of statements, where foo is a [label](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/label), not a key in an object literal.
이는 JavaScript가 화살표 다음에 오는 토큰이 왼쪽 괄호가 아닌 경우에만 화살표 함수를 표현식 본문이 있는 것으로 보기 때문에 괄호({}) 안의 코드는 명령문 시퀀스로 구문 분석되며, 여기서 foo 는 객체 리터럴의 키가 아니라 [레이블](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/label)입니다.

> To fix this, wrap the object literal in parentheses:
> 이 문제를 해결하려면 개체 리터럴을 괄호로 묶습니다.